erpnext/selling/doctype/crm_settings/crm_settings.json

```
@@ -0,0 +1,102 @@
+{
+ "actions": [],
+ "creation": "2021-09-09 17:03:22.754446",
+ "description": "Settings for Selling Module",
+ "doctype": "DocType",
+ "document_type": "Other",
+ "engine": "InnoDB",
+ "field_order": [
+  "campaign_naming_by",
+  "column_break_9",
+  "default_valid_till",
+  "section_break_5",
+  "allow_creation_of_leads_form_a_email",
+  "auto_creation_of_contact",
+  "create_event_on_next_contact_date",
+  "opportunity_section",
+  "close_opportunity_after_days",
+  "create_event_on_next_contact_date_opportunity"
+ ],
+ "fields": [
+  {
+   "fieldname": "campaign_naming_by",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Campaign Naming By",
+   "options": "Campaign Name\nNaming Series"
+  },
+  {
+   "fieldname": "default_valid_till",
+   "fieldtype": "Data",
+   "label": "Default Quotation Validity Days"
+  },
+  {
+   "fieldname": "column_break_9",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "15",
+   "description": "Auto close Opportunity after the no. of days mentioned above",
+   "fieldname": "close_opportunity_after_days",
+   "fieldtype": "Int",
+   "label": "Close Opportunity After Days"
+  },
+  {
+   "fieldname": "section_break_5",
+   "fieldtype": "Section Break",
+   "label": "Lead"
+  },
+  {
+   "default": "0",
+   "fieldname": "allow_creation_of_leads_form_a_email",
+   "fieldtype": "Check",
+   "label": "Allow Creation of Multiple Leads form a Email"
```

```suggestion
   "label": "Allow Creation of Multiple Leads from an Email"
```

**Allow Lead Duplication based on Emails** 

Is this better?